### DISCUSSION TITLE
Name of the Project

### DISCUSSION THREAD

**Discussion by Pitso-Paballo** [Pitso-Paballo/Innovate-Six#3] [Q&A] [answered]
As Discussed in our last meeting. An **Ai Assistant** Is to be created that will assist  toddlers enhance their reading and pronunciation skills by listening to them read aloud and giving them instant feedback **Any names that can be suggested for the project name?**

**Comment by Twala-TS** [comment, marked as answer]
"LearnMate" seems to be the most suitable name.

**Reply by Pitso-Paballo** [reply]
**Learnmate** it is perfect

**Reply by Twala-TS** [reply]
if we all cool with it then LearnMate will do.
